.buildkite/x-pack/pipeline.xpack.libbeat.yml

```diff
@@ -18,45 +14,176 @@ env:
   IMAGE_WIN_2019: "family/platform-ingest-beats-windows-2019"
   IMAGE_WIN_2022: "family/platform-ingest-beats-windows-2022"
 
-  #Packaging
-  PACKAGING_ARM_PLATFORMS: "linux/arm64"
-  PACKAGING_PLATFORMS: "+all linux/amd64 linux/arm64 windows/amd64 darwin/amd64 darwin/arm64"
-
   #Deps
   ASDF_MAGE_VERSION: 1.15.0
 
 steps:
+  - group: "Mandatory Tests"
+    key: "mandatory-tests"
```

Comment:
same comment for the group key, for consistency can we add the `x-pack/libbeat` prefix to this string, as done e.g. in https://github.com/elastic/beats/pull/38828/files#diff-4291f1a0414f4870861926dce4a866be0c14ce2b86f018e958458c53c719c7a2R27?